zebra-state/tests/basic.rs

```diff
@@ -27,7 +27,34 @@ static COMMIT_FINALIZED_BLOCK_MAINNET: Lazy<Vec<(Request, Result<Response, Trans
         ]
     });
 
-static COMMIT_FINALIZED_BLOCK_TESTNET: Lazy<Vec<(Request, Result<Response, TransError>)>> =
+static GET_TIP_ADD_REVERSED_TRANSCRIPT_MAINNET: Lazy<Vec<(Request, Result<Response, TransError>)>> =
+    Lazy::new(|| {
+        let block0: Arc<_> =
+            Block::zcash_deserialize(&zebra_test::vectors::BLOCK_MAINNET_GENESIS_BYTES[..])
+                .unwrap()
+                .into();
+        let block1: Arc<_> =
+            Block::zcash_deserialize(&zebra_test::vectors::BLOCK_MAINNET_1_BYTES[..])
+                .unwrap()
+                .into();
+        let hash0 = block0.as_ref().into();
+        let hash1 = block1.as_ref().into();
+        let height1 = block1.coinbase_height().unwrap();
+        vec![
+            // Insert the blocks in reverse order
+            (
+                Request::CommitFinalizedBlock { block: block1 },
+                Ok(Response::Committed(hash1)),
+            ),
+            (
+                Request::CommitFinalizedBlock { block: block0 },
+                Ok(Response::Committed(hash0)),
+            ),
+            (Request::Tip, Ok(Response::Tip(Some((height1, hash1))))),
+        ]
+    });
+
+static GET_TIP_ADD_REVERSED_TRANSCRIPT_TESTNET: Lazy<Vec<(Request, Result<Response, TransError>)>> =
     Lazy::new(|| {
         let block: Arc<_> =
             Block::zcash_deserialize(&zebra_test::vectors::BLOCK_TESTNET_GENESIS_BYTES[..])
```

Comment:
This test is named incorrectly - it only commits one block.

Instead, let's have tests for `COMMIT_FINALIZED_BLOCK_TESTNET` and `GET_TIP_ADD_REVERSED_TRANSCRIPT_TESTNET`, like the mainnet tests.

It might also be helpful to rename `block2` in the `COMMIT_FINALIZED_BLOCK_*` tests - it's actually a copy of the genesis block, not a block at height 2.